visitor/dumper.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/z7zmey/php-parser/walker"
 )
 
-// Dumper prints ast hierarchy to stdout
```

Comment:
It is correct to change `stdout` to `io.Writer`.

Dumper is expected any struct that implements io.Writer interface not only stdout.

Reply:
You want me to replace Writer with io.Writer ?

Reply:
Yep

Reply:
Done, if you want to have a look